ports/atmel-samd/usb.c

```diff
@@ -322,6 +331,9 @@ bool usb_connected(void) {
 void usb_cdc_background() {
     //
     if (mp_interrupt_char != -1 && cdc_enabled() && !pending_read) {
-        start_read();
+        // Make sure we have space in the buffer
+        if (usb_rx_count < 64) {
```

Comment:
Use USB_RX_BUF_SIZE here too and merge the condition with the outside if statement. No need to have two.